libs/checkpoint-postgres/Makefile

```diff
@@ -16,10 +16,10 @@ test:
 	EXIT_CODE=$$?; \
 	make stop-postgres; \
 	exit $$EXIT_CODE
-
+TEST ?= .
 test_watch:
 	make start-postgres; \
```

Comment:
would it make sense to test on both versions of postgres (15 and 16)? or is it an overkill?

Reply:
hm - maybe a good idea

Reply:
I think better for sep PR though